### PR TITLE
Adx 458 update tests

### PR DESCRIPTION
**Pull request by jsoppela**

### CODE REVIEW

File: ckanext/ytp/request/tests/test_plugin.py [hunk 1]
```diff
@@ -1,27 +1,38 @@
...
 from ckan.lib.helpers import url_for
 
+import ckan.model as model
+import ckanext.ytp.request.model as rmodel
+
 import ckan.tests.helpers as helpers
 import ckan.tests.factories as factories
 
-webtest_submit = helpers.webtest_submit
+#webtest_submit = helpers.webtest_submit
```
Comment (jonathansberry) [inline ckanext/ytp/request/tests/test_plugin.py]: Guessing this is a typo... Let's remove rather than comment out.

**Review by jonathansberry — CHANGES_REQUESTED** [review]
Yeah it's great. Just spotted a typo - we're commenting out a line rather than removing it.

File: ckanext/ytp/request/logic/action/update.py [hunk 2]
```diff
@@ -4,6 +4,7 @@
 from ckanext.ytp.request.mail import mail_process_status
 from ckan.lib.helpers import flash_success
 from ckan.common import _
+from sqlalchemy import asc, desc
```
Comment (Manoj-nathwani) [inline ckanext/ytp/request/logic/action/update.py]:
```suggestion
from sqlalchemy import desc
```

File: ckanext/ytp/request/logic/action/get.py [hunk 3]
```diff
@@ -3,6 +3,7 @@
 from ckan.lib.dictization import model_dictize
 from ckanext.ytp.request.model import MemberRequest
 from ckanext.ytp.request.helper import get_organization_admins
+from sqlalchemy import asc, desc
```
Comment (Manoj-nathwani) [inline ckanext/ytp/request/logic/action/get.py]:
```suggestion
from sqlalchemy import desc
```

File: ckanext/ytp/request/logic/action/delete.py [hunk 4]
```diff
@@ -4,6 +4,7 @@
 from ckan.lib.dictization import model_dictize
 from ckan.common import c
 from sqlalchemy.sql import func
+from sqlalchemy import asc, desc
```
Comment (Manoj-nathwani) [inline ckanext/ytp/request/logic/action/delete.py]:
```suggestion
from sqlalchemy import desc
```

- I see why you've aded `asc` here
- Buuuuut travis will fail as you're importing something which is never used
- You'll need to remove it to make travis happy

**Review by Manoj-nathwani — CHANGES_REQUESTED** [review]

**Comment by Manoj-nathwani** [comment]
Looks great, I've just removed the unused imports to stop travis being unhappy with pep8 validation checks. I remember it being suuuper strict 😅

**Comment by jonathansberry** [comment]
As we are migrating away from travis, and I have confirmed that these changes make the tests pass locally, I will merge and close this PR.

**Pull request merged**